Modules/Filtering/DisplacementField/include/itkTransformToDisplacementFieldFilter.hxx

```diff
@@ -203,11 +202,11 @@ TransformToDisplacementFieldFilter<TOutputImage, TParametersValueType>::Nonlinea
       // Compute corresponding input pixel position
       transformedPoint = transform->TransformPoint(outputPoint);
 
-      displacementPoint = transformedPoint - outputPoint;
+      const auto displacementVector = transformedPoint - outputPoint;
```

Comment:
The type isn't apparent from the right-hand side, so I'd put `VectorType` here instead of `auto`, where `using VectorType = ...` should be defined in the places where `PointType` is defined, and ultimately is `itk::Vector<double, 3>` in this case.

Reply:
@Leengit I would rather not add extra type aliases with this pull request. As a compromise, I just replaced this `auto` by `typename PointType::VectorType`, as I see that being used in similar cases, in the same source file. Hope it's OK to you!

Reply:
Looks good.